src/coreclr/scripts/genLttngProvider.py

```diff
@@ -115,6 +115,8 @@ def getLttngDataTypeMapping(runtimeFlavor):
         return coreCLRLttngDataTypeMapping
     elif runtimeFlavor.mono:
         return monoLttngDataTypeMapping
+    else:
+        return monoLttngDataTypeMapping
```

Comment:
```suggestion
    else:
        return portableLttngDataTypeMapping
```

Reply:
The mappinng is not Mono-specific. Ideally, we would switch CoreCLR to use the portable mapping too.